<!DOCTYPE html>
<html lang="en-us">
  <head><script src="/livereload.js?mindelay=10&amp;v=2&amp;port=1313&amp;path=livereload" data-no-instant defer></script>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
    
    <title>PhD school on Evolution in Computational Pangenomics | COPA 2024</title>
    <meta name="viewport" content="width=device-width,minimum-scale=1">
    <meta name="description" content="School on Evolution in Computational Pangenomics">
    <meta name="generator" content="Hugo 0.123.4">
    
    
    
    
      <meta name="robots" content="noindex, nofollow">
    

    
<link rel="stylesheet" href="/ananke/css/main.min.css" >



  
    <link rel="stylesheet" href="/css/custom.css">
  

    

    
      

    

    
    
      <link href="/index.xml" rel="alternate" type="application/rss+xml" title="COPA 2024" />
      <link href="/index.xml" rel="feed" type="application/rss+xml" title="COPA 2024" />
      
    

    
      <link rel="canonical" href="http://localhost:1313/">
    

    <meta property="og:title" content="PhD school on Evolution in Computational Pangenomics" />
<meta property="og:description" content="School on Evolution in Computational Pangenomics" />
<meta property="og:type" content="website" />
<meta property="og:url" content="http://localhost:1313/" />

<meta itemprop="name" content="PhD school on Evolution in Computational Pangenomics">
<meta itemprop="description" content="School on Evolution in Computational Pangenomics"><meta name="twitter:card" content="summary"/><meta name="twitter:title" content="PhD school on Evolution in Computational Pangenomics"/>
<meta name="twitter:description" content="School on Evolution in Computational Pangenomics"/>

	
  </head>

  <body class="ma0 avenir bg-near-white">

    

  
  <header class="cover bg-top" style="background-image: url('http://localhost:1313/imgs/como640.jpg');">
    <div class="bg-black-60">
      <nav class="pv3 ph3 ph4-ns" role="navigation">
  <div class="flex-l justify-between items-center center">
    <a href="/" class="f3 fw2 hover-white no-underline white-90 dib">
      
        COPA 2024
      
    </a>
    <div class="flex-l items-center">
      

      
        <ul class="pl0 mr3">
          
          <li class="list f5 f4-ns fw4 dib pr3">
            <a class="hover-white no-underline white-90" href="/pages/program/" title="Program page">
              Program
            </a>
          </li>
          
          <li class="list f5 f4-ns fw4 dib pr3">
            <a class="hover-white no-underline white-90" href="/speakers/" title="Speakers page">
              Speakers
            </a>
          </li>
          
          <li class="list f5 f4-ns fw4 dib pr3">
            <a class="hover-white no-underline white-90" href="/pages/location/" title="Location page">
              Location
            </a>
          </li>
          
        </ul>
      
      
<div class="ananke-socials">
  
</div>

    </div>
  </div>
</nav>

      <div class="tc-l pv4 pv6-l ph3 ph4-ns">
        <h1 class="f2 f-subheadline-l fw2 white-90 mb0 lh-title">
          PhD school on Evolution in Computational Pangenomics
        </h1>
        
          <h2 class="fw1 f5 f3-l white-80 measure-wide-l center mt3">
            School on Evolution in Computational Pangenomics
          </h2>
        
      </div>
    </div>
  </header>


    <main class="pb7" role="main">
      
 <article class="cf ph3 ph5-l pv3 pv4-l f4 tc-l center measure-wide lh-copy mid-gray">
    <p>The school is one of the network-wide events of the MSCA-RISE Project PANGAIA (Pan-genome Graph Algorithms and Data Integration) that is a European effort for guiding the transition from linear genomes to graph-based representations.
Classically, Genomes are strings over the letters A,C,G,T, which represent nucleotides, the building blocks of DNA. In view of ultra-large amounts of genome sequence data emerging from ever more and technologically rapidly advancing genome sequencing devices—in the meantime, amounts of sequencing data accrued are reaching into the exabyte scale—the driving, urgent question is:
how can we arrange and analyze these data masses in a formally rigorous, computationally efficient and biomedically rewarding manner?</p>
<p>After the success of the <em>Introduction to Pan-Genomics</em> school, this school will instead focus on evolutionary aspects of pangenomics. There are two aspects that will be discussed:</p>
<ul>
<li>How tumor, virus, and bacteria populations evolve during time, analyzing the computational aspects of different models of evolution.</li>
<li>How to analyze a pangenome to identify genetic variants associated with human diseases, distinguishing them from the plethora of neutral variants that arise from “healthy” evolution.</li>
</ul>
<p>In both cases, we will focus on graph-based frameworks, since one of the main allures of computational pangenomics is to get rid of linear reference genomes and the bias it introduces.
In fact, we remind that one of the main goals of the PANGAIA project is to put this shift of paradigm—from sequence- to graph-based representations of genomes —into full effect.</p>
<h1 id="key-dates">Key dates</h1>
<ul>
<li>Registration deadline: May 31st</li>
<li>School dates: September 8-12, 2024</li>
</ul>
<h1 id="sponsors">Sponsors</h1>
<p><a href="https://pangenome.eu"><img src="/imgs/PANGAIA-425.png" alt="Pangaia"></a></p>
<h1 id="school-directors">School Directors</h1>
<ul>
<li>Paola Bonizzoni (Univ. Milano-Bicocca, Italy)</li>
<li>Gianluca Della Vedova (Univ. Milano-Bicocca, Italy)</li>
<li>Alexander Schönhuth (Univ. Bielefeld, Germany)</li>
</ul>
<h1 id="organizers">Organizers</h1>
<ul>
<li>Paola Bonizzoni</li>
<li>Simone Ciccolella</li>
<li>Gianluca Della Vedova</li>
<li>Luca Denti</li>
</ul>

  </article>
  

    </main>
    

  </body>
</html>
